### ISSUE TITLE
ui: could not visualized right when target is too many.

### ISSUE THREAD

**Issue by zdyj3170101136** [parca-dev/parca#1723]
I have about thousands of targets.

with or without  query filter,  the ui is white.
![截屏2022-09-15 下午4 04 53](https://user-images.githubusercontent.com/52686065/190349909-429ab886-578d-41a1-b215-5f59d6767358.png)

#### version
parca v0.12.1
#### expect behaviour
the ui could visualize right, with thousnads of lines.

**Comment by zdyj3170101136** [comment]
after upgrade the  --storage-active-memory from 512mb to 50gb, seems work right

**Comment by brancz** [comment]
Yeah, that seems like it works as expected, as when the `--storage-active-memory` is full it is rotated out.
